stories/Textarea.tsx

```diff
@@ -49,9 +55,24 @@ export const textareaAutosizeKnobs = (tabName = 'Textarea'): any => {
 
 storiesOf('Textarea', module)
   .addDecorator(withKnobs)
+  .add('uncontrolled', () => <Textarea {...textareaKnobs()} {...textKnobs()} {...boxKnobs()} />)
+  .addDecorator(StateDecorator(store))
   .add('with defaults', () => (
-    <Textarea {...textareaKnobs()} {...textKnobs()} {...boxKnobs()} defaultValue="Textarea Text" />
+    <Textarea
+      {...textareaKnobs()}
+      {...textKnobs()}
+      {...boxKnobs()}
+      value={store.get('value')}
+      onChange={(value: any) => store.set({ value })}
```

Comment:
I know it's a story, but we could change the type here to string since we know it.